scripts/set_gradle_proxy.sh

```diff
@@ -53,7 +53,7 @@ writeProxyStringToGradleProps () {
 }
 
 writeProxyEnvToGradleProps () {
-    local GRADLE_PROPS="$HOME/.gradle/gradle.properties"
+    local GRADLE_PROPS="$GRADLE_USER_HOME/gradle.properties"
```

Comment:
Commit message nits:
- Just use the literal filename (`set_gradle_proxy.sh`) as the prefix in the title.
- In exchange, drop "set_gradle_proxy.sh" from the sentence in the commit message body.- s/docker/Docker/
- s/gradle/Gradle/
- s/GRADLE_USER_HOME/$GRADLE_USER_HOME/
- Consider putting "quoted code" (environment variables, exception names) into backticks like in Markdown.

Reply:
Done

Reply:
It just occurred to me that this script should also work outside of the `Dockerfile` context, where the `GRADLE_USER_HOME` variable might not be set. So we should probably use `${GRADLE_USER_HOME:-$HOME/.gradle}` here.